content/guides/embed/ui-elements/installation.md

```diff
@@ -200,7 +200,7 @@ For example, if you self-host using a `box-assets` directory, these URLs must
 be accessible:
 
 - `https://cdn.YOUR_SITE.com/box-assets/2.81.0/en-US/preview.js`
-- `https://cdn.YOUR_SITE.com/box-assets/third-party/text/0.116.0/papaparse.min.js`
+- `https://cdn.YOUR_SITE.com/box-assets/third-party/text/2.65.0/papaparse.min.js`
```

Comment:
The latest version for papaparse for box-content-preview is `2.65.0`: https://github.com/box/box-content-preview/tree/master/src/third-party/text